src/Units/UnitModel.java

```diff
@@ -83,18 +83,19 @@ public static String standardizeID(String input)
   public final int fuelBurnIdle;
   @Builder.Default public final int fuelBurnPerTile = 1;
   @Builder.Default public final int maxMaterials = 0;
-  @Builder.Default public final boolean needsMaterials = true;
+  @Builder.Default public final boolean needsMaterials = true; // Unit has a finite materials count, which is likely 0.
   public final int visionRange;
-  @Builder.Default public final boolean visionPierces = false;
-  @Builder.Default public final boolean hidden = false;
-  public final Set<TerrainType> healableHabs = new HashSet<TerrainType>();
+  @Builder.Default public final boolean visionPierces = false; // Can see into cover terrain in fog of war.
+  @Builder.Default public final boolean hidden = false; // Invisible even outside of fog, in any terrain; usually on on a variant of a unit type that transforms.
```

Comment:
"on on"

Reply:
This isn't a typo.
"(Usually on) on a variant of a unit type that transforms."
I could use a less ambiguous word, but then I wouldn't have "on on".

Reply:
I... uh. I mean, I'd suggest at least adding a comma after the first "on", but I suspect you'd object because then it wouldn't have "on on". Whatever, it's a comment, and at least intelligible enough as is.

Reply:
I wanted to say it wouldn't grammatically fit, but it actually would.
Fair enough.